### ISSUE TITLE
Export Document expansion utilities

### ISSUE THREAD

**Issue by magnustroest** [jdpnielsen/uda-to-typescript#3]
This would be helpful for actually consuming types together with the fetcher functions.

**Comment by jdpnielsen** [comment]
Yes! I exported some of the current helpers, but i'll look at implementing and documenting more.
